repository.yaml

```diff
@@ -9,4 +9,10 @@ leakDetectionExemptions:
        - '/app/utils/model/rows/IndividualModelRows.scala'
        - '/test/forms/arrangement/DoYouKnowTheReasonToReportArrangementNowFormProviderSpec.scala'
        - '/test/forms/individual/IsIndividualResidentForTaxOtherCountriesFormProviderSpec.scala'
-       - '/test/forms/organisation/WhatAreTheTaxNumbersForNonUKOrganisationFormProviderSpec.scala'
\ No newline at end of file
+       - '/test/forms/organisation/WhatAreTheTaxNumbersForNonUKOrganisationFormProviderSpec.scala'
+    text:
```

Comment:
Unfortunately `text` is a single string.  If you want to keep the exemptions in `repository.yaml` you'll need to define it like so:
```
leakDetectionExemptions:
  - ruleId: 'aws_secret_access_key'
    filePaths:
       - '/app/utils/rows/IndividualRows.scala'
       - '/app/utils/model/rows/IndividualModelRows.scala'
       - '/test/forms/individual/IsIndividualResidentForTaxOtherCountriesFormProviderSpec.scala'
    text: 'isIndividualResidentForTaxOtherCountries'
  - ruleId: 'aws_secret_access_key'
    filePath: '/app/utils/rows/ReporterRows.scala'
    text: 'reporterOrganisationEmailAddressQuestion'
  - ruleId: 'aws_secret_access_key'
    filePath: '/test/forms/arrangement/DoYouKnowTheReasonToReportArrangementNowFormProviderSpec.scala'
    text: 'doYouKnowTheReasonToReportArrangementNow'
  - ruleId: 'aws_secret_access_key'
    filePath: '/test/forms/organisation/WhatAreTheTaxNumbersForNonUKOrganisationFormProviderSpec.scala' 
    text: 'whatAreTheTaxNumbersForNonUKOrganisation'
```